src/components/panel/panel.spec.js

```diff
@@ -77,6 +77,9 @@ describe('$mdPanel', function() {
 
     // TODO(ErinCoughlan) - Remove when close destroys.
     panelRef = null;
+
+    // If just attachPanel is called w/o close, this cleans things up.
+    angular.element(document.querySelector(PANEL_WRAPPER_CLASS)).remove();
```

Comment:
We call close a few lines above this to make sure it's cleaned up. If any element is added to the DOM, it gets added to the attachedElements list, which are all removed here as well.